README.md

```diff
@@ -19,13 +19,18 @@ compatible with certain Vitess and Kubernetes versions, as shown in this table:
 | `v2.11.*`               | `v18.0.*`                   | `v1.22.*`, `v1.23.*`, `v1.24.*`, or `v1.25.*` |
 | `v2.12.*`               | `v19.0.*`                   | `v1.25.*`, `v1.26.*`, `v1.27.*`, or `v1.28.*` |
 | `v2.13.*`               | `v20.0.*`                   | `v1.25.*`, `v1.26.*`, `v1.27.*`, or `v1.28.*` |
-| `latest`                | `latest`                    | `v1.25.*`, `v1.26.*`, `v1.27.*`, or `v1.28.*` |
+| `latest`                | `latest`                    | `v1.29.*`, `v1.30.*`, `v1.31.*`               |
```

Comment:
Don't we need an overlapping Kubernetes version with the previous release, to make upgrading easier/less risky?

Reply:
That is, to support v1.28 through v1.31.

Reply:
The main branch does support 1.28 technically